content/security/team.adoc

```diff
@@ -47,7 +47,7 @@ In order to join, you'll need to have:
 . a link:https://accounts.jenkins.io/[Jenkins community account] and have logged in to Artifactory and Jira before,
 . a link:https://github.com[GitHub] account with link:https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/[two-factor authentication enabled],
 . a link:https://github.com/jenkinsci/infra-cla/[Contributor License Agreement] (CLA) in place, and
-. a link:https://freenode.net/kb/answer/registration[registered nickname on Freenode IRC].
+. a link:https://libera.chat/guides/registration[registered nickname on Libera Chat].
```

Comment:
@danielbeck what's the plan here? are we re-adding the cert channel?

do you still require an IRC nickname registered?

Reply:
IIUC the plan was to drop the CERT channel in favor of mailing lists

Reply:
As that isn't publicly documented it doesn't really matter anyway for this.